client/utils/challengeTypes.js

```diff
@@ -70,6 +70,6 @@ exports.submitTypes = {
 exports.helpCategory = {
   [html]: 'HTML-CSS',
   [js]: 'JavaScript',
```

Comment:
@QuincyLarson Will these automatically apply the category `Curriculum Help` and the applicable subcategory?